tests/data/semparse/action_space_walker_test.py

```diff
@@ -47,4 +48,10 @@ def test_get_logical_forms_with_agenda(self):
                                                                                 '<o,o> -> touch_wall'])
         # Permutations of the three functions. There will not be repetitions of any functions
         # because we limit the length of paths to 10 above.
-        assert len(black_triangle_touch_forms) == 6
+        assert set(black_triangle_touch_forms) == \
```

Comment:
Readability nit: backslashes like this are annoying, and I'd avoid them wherever possible.  Not a big deal at all, but I think it looks nicer if you either define a variable like `expected_logical_forms` above and have the assert use that, or define the `set` inline, but keep the `set([` on the first line and do wrapping without needing the backslash.

Reply:
Done.